code/datums/traits/good.dm

```diff
@@ -29,6 +29,22 @@
 
 
 
+/datum/quirk/buster
+	name = "Buster"
+	desc = "Old fights have made you able to do more damage, although pretty inaccurately."
```

Comment:
You have a history of fighting unarmed, allowing you to do more raw damage with your fists. You tend to weaken less often, however.